pallets/sp-mvm/src/lib.rs

```diff
@@ -90,6 +91,7 @@ pub mod pallet {
     /// - Value: `WriteSet` as bytes
     // TODO: Experimentally try hasher [Identity][frame_support::Identity]
     //                           because key are already encoded - hashes.
+    /// SBP M1 What would you expect from this change?
```

Comment:
This is a possible optimization because we already have hashes as keys and `Identity` is no-op hasher. So the idea was in use our keys as is without probably unnecessary extra hashing. But we are not tried and tested this yet.

Reply:
Right I am just unclear how big the gain would be. Better double check this would be safe :)